lib/usbd/backend/usbd_stm32_otg_fs.c

```diff
@@ -75,10 +79,10 @@ static usbd_device *init(const usbd_backend_config *config)
 	_usbd_dev.config = config;
 
 	if (config->feature & USBD_VBUS_SENSE) {
-		if (OTG_FS_CID >= 0x00002000) { /* 2.0 */
+		if (OTG_FS_CID >= 0x00002000) { /* 2.0 HS core*/
```

Comment:
heh. my version of RM0385 says:
Reset value: 0x0000 3000 for USB OTG FS
Reset value: 0x0000 3100 for USB OTG HS
looks like this code need target MCU specification - it depends on MCU family. And even more - look like this code not aplicatable on stm32f7 family 8/